docs/tests/e2e-tests.md

```diff
@@ -0,0 +1,103 @@
+---
+title: End to End Tests
+---
+
+# End to End Tests
+
+[Cypress](https://www.cypress.io), a JavaScript based framework, is used for end
+to end (E2E) testing. Tests and associated utilities are within the `cypress`
+directory.
+
+```shell
+$ tree app/javascript/article-form -L 1
+cypress
+├── fixtures
+|   └── initialAdmin.json
+|   └── ...
+├── integration
+|   └── **/*.spec.js
+├── plugins
+└── support
```

Comment:
Does this need updated? The `tree` command is pointing to the `article-form` dir rather than the cypress one, and the fixtures/integration dirs are now organised with sub directories

Reply:
Done. I just ran the command `tree cypress` to get the actual output instead of hacking a pre existing one. 🙃